mesa/Person.py

```diff
@@ -188,10 +188,7 @@ def remove_professional(self, forlorn): self.remove_link(self, forlorn, 'profess
 
     def age_between(self, low, high):
         return self.age() >= low and self.age() < high
-    
-    def family(self): # maybe add self?
```

Comment:
Changed with Person.family_link_neighbors(), I created this function and realized afterwards that it already existed!